ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -8,3 +8,16 @@ def ifft(a, n=None, axis=-1, norm=None):
     if norm is None:
         norm = "backward"
     return ivy.ifft(a, axis, norm=norm, n=n)
+
+@to_ivy_arrays_and_back
+def ifft2(a,
+          n=None,
+          axis=(-1,-2),
+          norm=None
+          ):
+    a= ivy.array(a, dtype=ivy.complex128)
+    if norm is None:
+        norm = "backward"
+    return ivy.ifft2(a,axis, norm=norm, n=n)
```

Comment:
Hey! I just realised this, the `ifft2` function is not present in Ivy, you can either propose this function to be included in the Ivy API and implement it yourself if you'd like, or in other cases you should implement this frontend function as compositional function by using the other functions. More information here: https://lets-unify.ai/ivy/overview/contributing/open_tasks.html#frontend-apis

thanks!

Reply:
done @juliagsy thanks for your continuous help. any other recommendations ?